### ISSUE TITLE
why loss go up?

### ISSUE THREAD

**Issue by jovahe** [ternaus/kaggle_dstl_submission#21]
when i use jaccard as loss function, the training loss dose not go down but go up. what is wrong with me?
